applegacy/src/main/AndroidManifest.xml

```diff
@@ -26,4 +26,17 @@
         </activity>
         <activity android:name="com.hamdan.forzenbook.legacy.createaccount.LegacyCreateAccountActivity" />
     </application>
-</manifest>
\ No newline at end of file
+</manifest>
+
+<!--<activity-->
```

Comment:
Why leave here?

Reply:
yea its not needed, I should've put a todo for when the legacy post is finished to put it back, but honostly its not needed to easily put it back.